docs/ide-usage.md

```diff
@@ -53,6 +53,14 @@ The following plugins are needed:
      * ORM Enhancement: `App Engine → ORM` → clear all the entries, and add the following entry: `src/main/java/teammates/storage/entity/*.java`.
      * Validation exclusion: `App Engine → Validation` → add two entries: `src/test/java` and `src/client/java`.
      * WAR directory: `Web Application` → tick both `This project has a WAR directory` and `Launch and deploy from this directory`, and enter `src/main/webapp` as `WAR directory`.
+   * JDK: `Java Build Path → Libraries` → ensure that the system library used is JDK 7.
+
+       ![setupguide-7.png](images/setupguide-7.png)
```

Comment:
Oops, apologies for the oversight 😅 I managed to use a custom resolution setting through a third party tool to get a screenshot of the same resolution but alas, I realised that screenshot actually contains our home directory username:
![screen shot 2017-04-17 at 1 04 45 am](https://cloud.githubusercontent.com/assets/6811428/25072806/c645fd80-230a-11e7-8836-2bb920f2db67.png)

Hence, it'd be best to use @wkurniawan07's for consistency. @wkurniawan07 could you please provide a screenshot 😅